### ISSUE TITLE
Is there a way to disable horizontal scroll?

### ISSUE THREAD

**Issue by chardskarth** [neovide/neovide#1609]
**Is your feature request related to a problem? Please describe.**
I don't find it helpful that I'm able to scroll horizontally. It's useless when I see the whole line anyway.

**Describe the solution you'd like**
Is there a way to disable horizontal scroll?
Or atleast make vertical scrolling ergonomic enough and not detect a useless horizontal scroll.

I'm playing around 
let g:neovide_touch_deadzone = 6.0
let g:neovide_touch_drag_timeout = 1

But haven't found a way to "disable" it.

**Describe alternatives you've considered**

**Additional context**
https://gfycat.com/ripesharpindianpangolin

**Comment by MultisampledNight** [comment]
Thanks for reporting!

#1511 also wanted to include a change in that regard, but it has other issues which make touch scrolling pretty uncomfortable.

But reword what you want, you basically want the touch scrolling to "lock" into a horizontal/vertical movement, optionally? What would be your desired way of configuring detection for those, if at all?

**Comment by chardskarth** [comment]
Yep! I think it's helpful to have a global setting that locks vertical scrolling (something like `g:neovide_vertical_scroll_lock`).

I'll have this set to true by default and have a key that toggles it off. @MultisampledNight

**Comment by ad-chaos** [comment]
You can solve this from within neovim by disabling the left and right scroll wheels and use shift+scrollwheelup/down instead
```vim
nnoremap <ScrollWheelRight> <Nop>
nnoremap <ScrollWheelLeft> <Nop>
nnoremap <S-ScrollWheelUp> <ScrollWheelRight>
nnoremap <S-ScrollWheelDown> <ScrollWheelLeft>
```

**Comment by chardskarth** [comment]
Welp, this works. Thanks much. Closing now.
